src/subsystems/haskell/translators/utils.nix

```diff
@@ -0,0 +1,57 @@
+{
+  lib,
+  pkgs,
+}: let
+  l = lib // builtins;
+
+  all-cabal-json = let
```

Comment:
We should probably manage this as a flake input as seen in https://github.com/nix-community/dream2nix/pull/174/files